apps/settings/locales/settings.en-US.properties

```diff
@@ -63,9 +65,9 @@ shortStatus-connected        = Connected
 shortStatus-connectingfailed = Connection failed
 shortStatus-disconnected     = Offline
 wifi-sleep=Wi-Fi Sleep
-wifi-sleep-description=Enable to automatically turn off Wi-Fi during sleep to save battery power.  
+wifi-sleep-description=Enable to automatically turn off Wi-Fi during sleep to save battery power.
 manageNetworks=Manage Networks
-wifi-bad-credentials-title=Failed connecting to Wi-Fi
+wifi-bad-credentials-title=Failed Connecting to Wi-Fi
```

Comment:
Even capitalization changes can't be made without changing the key.  Our UX team has always preferred lower-case for strings like this, however. If the Litehouse specs say otherwise, please double-check with Mozilla's UX team before making string changes like that.